### PR TITLE
Int gui feb18

### PR DESCRIPTION
**Pull request by Trotyl15**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "common"]
 	path = common
-	url = git@github.com:UWARG/common.git
```
Comment (m6keller) [inline .gitmodules:3]: @Trotyl15 I think that the url should stay with ssh url because i htink this is what most people use

**Comment by m6keller** [comment]
@Trotyl15 I think it looks good I just think that the .gitmodules should use the ssh url instead of the https:// one but its not a big deal

**Comment by Trotyl15** [comment]
> 

oh cuz the ssh url didn't work for some people, that's why I changed it

**Comment by m6keller** [comment]
> @m6keller oh cuz the ssh url didn't work for some people, that's why I changed it

ok sounds good then im good to merge

**Comment by Trotyl15** [comment]
@m6keller can you solve the conflict in the ground_recieve.py? Im not sure if you changed any of these

**Comment by m6keller** [comment]
@Trotyl15 I resolved the merge conflicts but there were a lot of conflicts. do u mind looking thru to make sure nothing looks super off? I ran with the MockGroundReceive module and the GUI runs fine

**Pull request merged**